qa/ccs-rolling-upgrade-remote-cluster/build.gradle

```diff
@@ -20,8 +20,8 @@ dependencies {
 }
 
 for (Version bwcVersion : BuildParams.bwcVersions.wireCompatible) {
-  String baseName = "v${bwcVersion}"
```

Comment:
This solves a problem with for loops and lazy evaluated strings that causes late resolved GStrings to only contain the last bwcVersion of the for loop

Reply:
Ug, I've been bitten by that so many times.